client/world.js

```diff
@@ -9,6 +9,8 @@ var World = function (chunkSize_, renderRadius_) {
   // how far do you have to move from the focused chunk to change focus?
   var chunkRadius = 2;
 
+  var worldSize = {x: 0, y: 0, z: 0}
```

Comment:
if u use the random chunk selection function I made, you don't need this anymore - I would say remove it

Reply:
Removed after adding your function.